pubnub-chat-impl/src/commonMain/kotlin/com/pubnub/chat/internal/message/BaseMessage.kt

```diff
@@ -375,23 +383,23 @@ abstract class BaseMessage<T : Message>(
         internal fun filterAction(actions: Actions?, action: PNMessageAction): Actions {
             return buildMap {
                 actions?.entries?.forEach { entry ->
-                    put(
-                        entry.key,
-                        buildMap {
-                            entry.value.forEach { innerEntry ->
-                                if (entry.key == action.type && innerEntry.key == action.value) {
-                                    put(
-                                        innerEntry.key,
-                                        innerEntry.value.filter {
-                                            it.actionTimetoken != action.actionTimetoken || it.uuid != action.uuid
-                                        }
-                                    )
-                                } else {
-                                    put(innerEntry.key, innerEntry.value)
+                    val actionMap = buildMap {
+                        entry.value.forEach { innerEntry ->
```

Comment:
What is a reason for this change?

Reply:
https://github.com/pubnub/js-chat/pull/190